src/client/java/teammates/client/scripts/scalabilitytests/InstructorFeedbackResultsPageScalabilityTest.java

```diff
@@ -18,20 +33,196 @@
 
     private static final String DATA_FOLDER_PATH = "src/client/java/teammates/client/scripts/scalabilitytests/data/";
     private static final Logger log = Logger.getLogger();
+    private static DataBundle testDataMax;
 
     @Override
     protected void prepareTestData() {
         // the actual test data is refreshed before each test method
     }
 
     private void refreshTestData(int numStudents, int numQuestions) {
```

Comment:
I'd actually recommend closing this PR and starting afresh, as this one is getting too long (plus the approach has completely changed).